inmemorystorage/storage.py

```diff
@@ -91,11 +91,12 @@ def __init__(self, dirs=None, files=None, parent=None):
         self.parent = parent
 
     def resolve(self, path, create=False, use_bytes=False):
-        path_bits = path.strip('/').split('/', 1)
+        path = os.path.normpath(path)
```

Comment:
This is needed because the `path` isn't guaranteed to be in the Unix format (`/` as the separator).

Django will ensure that the `path` is in the Unix format _after it hits the database_, but it can be passed in as the Windows format when saving the file.